everware/spawner.py

```diff
@@ -21,14 +21,14 @@
 
 from escapism import escape
 
-import git
-
 from .image_handler import ImageHandler
 
 import ssl
 
 import json
 
+from .git_executor import GitExecutor
```

Comment:
Can you move this to be with the `ImageHandler` import and also move the `ssl` and `json` imports to the top (they come from the stdlib so shouldn't be down here)